daemon/container.go

```diff
@@ -896,21 +896,21 @@ func (container *Container) Copy(resource string) (io.ReadCloser, error) {
 		return nil, err
 	}
 
-	basePath, err := container.getResourcePath(resource)
-	if err != nil {
+	if err := container.mountVolumes(); err != nil {
 		container.Unmount()
```

Comment:
Probably more of a 'go' question, but is there any reason these aren't in "defer"s ?